core/src/commonMain/kotlin/com/sourcepoint/mobile_core/network/requests/CustomConsentRequest.kt

```diff
@@ -0,0 +1,12 @@
+package com.sourcepoint.mobile_core.network.requests
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CustomConsentRequest (
+    val consentUUID: String,
+    val propertyId: Int,
+    val vendors: List<String> = emptyList(),
+    val categories: List<String> = emptyList(),
+    val legIntCategories: List<String> = emptyList()
+)
```

Comment:
I don't see a case in which `CustomConsentRequest` will be instantiated _without_ passing values to `vendors`, `categories` or `legIntCategories`. Because of that, I think it's ok not to give them default values, in other words, I think we should remove the ` = emptyList()`.